contrib/android/src/python/pants/contrib/android/tasks/unpack_libraries.py

```diff
@@ -155,7 +155,7 @@ def create_android_library_target(self, binary, library, coordinate, unpacked_aa
 
     # Depending on the contents of the unpacked aar file, create the dependencies.
     deps = []
-    if os.path.isdir(resource_dir):
+    if os.path.isdir(resource_dir) and os.listdir(resource_dir):
```

Comment:
Hrm, not sure this is right. What if an `.aar` contained just `res/empty_dir/`? The full recursive glob used by the syntetic library may need to be eagerly evaluted to test for the presence of >0 _files_. I'll try to test this case to see if that is needed.

Reply:
This is looking solid enough for review. PTAL.